src/app/component/user/components/profile/users-achievements/users-achievements.component.scss

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat&display=swap');
```

Comment:
How it related to testing?

Reply:
I split user's friends and achievements into two components, because there is different logic